apps/apps.c

```diff
@@ -2432,10 +2432,18 @@ BIO *dup_bio_out(int format)
 {
     BIO *b = BIO_new_fp(stdout,
                         BIO_NOCLOSE | (istext(format) ? BIO_FP_TEXT : 0));
+    void *prefix = NULL;
+
 #ifdef OPENSSL_SYS_VMS
     if (istext(format))
         b = BIO_push(BIO_new(BIO_f_linebuffer()), b);
 #endif
+
+    if ((prefix = getenv("HARNESS_OSSL_PREFIX")) != NULL) {
```

Comment:
Most other variables are HARNESS_xxx; only one is HARNESS_OSSL_xxx.  Can we not add to the OSSL list?

Reply:
We already have `HARNESS_OSSL_LEVEL`, and I frankly prefer to claim the `HARNESS_OSSL_` namespace to avoid possible clashes with the `HARNESS_` namespace *which is not ours, it's part of [Test::Harness](https://perldoc.perl.org/5.8.8/Test/Harness.html)* and all its more modern dervates.  `HARNESS_OSSL_` is a safe space to use.

Reply:
So those are all standard HARNESS variables?  Makes sense, thanks.